.github/workflows/rollback.yml

```
@@ -0,0 +1,144 @@
+            echo "sam_config_name=prod" >> $GITHUB_OUTPUT
+          elif [ "${{ github.event.inputs.target_environment }}" == "development" ]; then
+            echo "sam_config_name=dev" >> $GITHUB_OUTPUT
+          else
+            echo "sam_config_name=test" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Validate Rollback Commit
+        id: validate_commit
+        run: |
+          COMMIT="${{ github.event.inputs.rollback_commit }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)